policy/modules/roles/sysadm.te

```diff
@@ -71,6 +71,10 @@ ifndef(`enable_mls',`
 	logging_run_auditctl(sysadm_t, sysadm_r)
 ')
 
+ifdef(`init_systemd',`
+	init_dbus_chat(sysadm_t)
```

Comment:
Please add a comment for why this is needed. This access otherwise seems too generic and is unclear why it would be necessary.